src/dispatch/messaging.py

```diff
@@ -49,6 +50,13 @@ class MessageType(str, Enum):
 
 INCIDENT_TASK_LIST_DESCRIPTION = """The following are open incident tasks."""
 
+DOCUMENT_EVERGREEN_REMINDER_DESCRIPTION = """
+You are the owner of the following incident documents.
+This is a reminder that these documents should be kept up to date in order to effectively
+respond to incdients. Please review them and update or deprecate the documents as appropriate.""".replace(
```

Comment:
```suggestion
respond to incidents. Please review them and update, or clearly mark the documents as deprecated.""".replace(
```

Reply:
My thinking was that if we don't yet have a Dispatch-native way to mark these as deprecated, we should hint that they should mark them as deprecated in the document itself.